catalog/spatial/wfs/spatial-wfs-featuretransformer-xstream/pom.xml

```diff
@@ -133,50 +114,25 @@
       <plugin>
         <groupId>org.apache.felix</groupId>
         <artifactId>maven-bundle-plugin</artifactId>
+        <extensions>true</extensions>
         <configuration>
           <instructions>
+            <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+            <Export-Package/>
```

Comment:
✏️ Super nitpick: you can remove the empty `<Export-Package>` element.

Reply:
By default all packages will be exported so you need an empty tag if you don't want to export anything.